actions/gradle/publish/action.yml

```diff
@@ -38,30 +50,42 @@ runs:
     - name: Set up JDK
       uses: actions/setup-java@5ffc13f4174014e2d4d4572b3d74c3fa61aeb2c2 # v3.11.0
       env:
-        MAVEN_USERNAME: ${{ secrets.maven-username }}
-        MAVEN_PASSWORD: ${{ secrets.maven-password }}
-        GPG_KEY_PASS: ${{ secrets.gpg-key-pass }}
+        MAVEN_USERNAME: ${{ inputs.maven-username }}
+        MAVEN_PASSWORD: ${{ inputs.maven-password }}
+        GPG_KEY_PASS: ${{ inputs.gpg-key-pass }}
       with:
         distribution: temurin
         java-version: ${{ inputs.jdk-version }}
         server-id: ossrh
         server-username: MAVEN_USERNAME
         server-password: MAVEN_PASSWORD
-        gpg-private-key: ${{ secrets.gpg-private-key }}
+        gpg-private-key: ${{ inputs.gpg-private-key }}
         gpg-passphrase: GPG_KEY_PASS
+    - name: Download the slsa attestation
+      uses: slsa-framework/slsa-github-generator/.github/actions/secure-download-folder@v1.8.0-rc.1
```

Comment:
reference `@main` (we can't merge until the RC release is completed or reverted). Same comment for line 72